community/lxcfs/APKBUILD

```diff
@@ -7,7 +7,7 @@ pkgdesc="FUSE filesystem for LXC"
 url="https://linuxcontainers.org/lxcfs"
 arch="all"
 license="Apache 2.0"
-depends="cgmanager shadow"
+depends="cgmanager shadow shadow-uidmap uidmapshift"
```

Comment:
Does it really need `shadow`, isn’t `shadow-uidmap` sufficient? I’ve created `shadow-uidmap` subpackage specially for LXC, to avoid installing full `shadow` package when I need just `newuidmap` and `newgidmap`.